petastorm/namedtuple.py

```diff
@@ -0,0 +1,202 @@
+# Only Python 3.x needs a special namedtuple implementation for > 255 values.
+
+from __future__ import print_function
+
+import sys as _sys
+from collections import namedtuple
+from keyword import iskeyword as _iskeyword
+import six
+
+
+if six.PY2:
+    namedtuple_gt_255_fields = namedtuple
+else:
+    _class_template = """\
```

Comment:
Apparently Python 3.7 also supports >255 namedtuple fields. Let's use a standard namedtuple for 3.7.

Reply:
Indeed, if I remember well, the fields limit is maxint in python 3.7. I'll limit the use to Python 3.0.x to 3.6.x.